ingest/defaults/config.yaml

```diff
@@ -13,24 +13,60 @@ ncbi_taxon_id: ""
 # The list of NCBI Datasets fields to include from NCBI Datasets output
 # These need to be the mneumonics of the NCBI Datasets fields, see docs for full list of fields
 # https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/dataformat/tsv/dataformat_tsv_virus-genome/#fields
+#
+# The default includes all available fields to be able to easily see the
+# uncurated metadata by running the workflow with the target `data/ncbi_dataset_report.tsv`
+# Remove any fields that are not needed in your workflow to reduce file size and save space.
 # Note: the "accession" field MUST be provided to match with the sequences
 ncbi_datasets_fields:
```

Comment:
Another solution (the one I was assuming) is switch the `format_ncbi_dataset_report` rule to use the command from your ingest tutorial:

```
    $ dataformat tsv virus-genome \
        --package ingest/data/ncbi_dataset.zip \
        > ingest/data/raw_metadata.tsv
```

And then use `format_ncbi_datasets_ndjson` to prune the fields down to the desired `ncbi_datasets_fields`, change their casing (etc). I presume that NCBI will add fields to their datasets over time and it'd be a pain to miss important fields because we weren't aware of them (i.e. hadn't realised they existed because they were missing from our `ncbi_datasets_fields`)

Reply:
Yeah, both the mnemonics (with hyphens) and human readable names (with spaces) of the NCBI fields don't work well with our usual tools for TSV manipulation (e.g tsv-utils). So pruning the fields down and renaming them after the `dataformat` call will require a separate script that I don't really think it's worth it.

Reply:
Hmm. Ok. How about adding another rule and then then we can ask people to invoke it via `nextstrain build ingest dump_ncbi_dataset_report` or somesuch?

```
rule dump_ncbi_dataset_report:
    input:
        dataset_package="data/ncbi_dataset.zip",
    output:
        ncbi_dataset_tsv="data/ncbi_dataset_report_raw.tsv",
    shell:
        """
        dataformat tsv virus-genome \
            --package {input.dataset_package} > {output.ncbi_dataset_tsv}
        """
```


(I say this because I think it's inevitable that we'll miss new NCBI fields in the future because we think we're looking at all the data but really we're just looking at a subset which our config's defined. The measles strain name for instance.)

Reply:
Good idea, similar to @j23414' proposal in https://github.com/nextstrain/pathogen-repo-guide/issues/30! 

I'll update this PR with the new rule and update the tutorial to use it!

Reply:
Updated in https://github.com/nextstrain/pathogen-repo-guide/pull/38/commits/aa67664c4c7e762b65fb8b168c350880230b7e5d